### DISCUSSION TITLE
knowledge : where are the model files stored ?

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [open-webui/open-webui#528] [General]
Where are the actual model files stored on the disk for ubuntu os?

**Comment by justinh-rahb** [comment]
Depends on how you installed Ollama, but it's addressed in their FAQ: https://github.com/jmorganca/ollama/blob/main/docs/faq.md#where-are-models-stored

>## Where are models stored?
>* macOS: ~/.ollama/models.
>* Linux: /usr/share/ollama/.ollama/models
